testData/testprojects/intellij-integration/src/java/org/pantsbuild/testproject/simple/BUILD

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+jvm_binary(name = 'simple',
```

Comment:
let's put it to 

testData/testprojects/intellij-integration/src/java/org/pantsbuild/testproject/failures/simple

folder